CapybaraLearning/pages/dashboard_part.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../all_requires.rb'
+require_relative 'menu_part'
+
+module DashbordPart SitePrism::Page
+  def get_main_page_ref
+    all(:xpath, "//div[contains(@id,'wpbody-content')]//a")
+  end
+
+  def get_main_page_buttons
+    all(:css, 'button')
+  end
+
+  def get_main_page_headings
+    headings = Array.new
```

Comment:
you can initialize array just like this -  headings = []
also, you can try to iterate over array of selectors ['h1', 'h2', 'h3'] to get all headings.
another tip - string arrays in ruby can be written in the following format - w%(h1 h2 h3)